packages/app/test/firebaseApp.test.ts

```diff
@@ -15,22 +15,16 @@
  * limitations under the License.
  */
 
```

Comment:
Should this file be renamed? I don't think it's testing the `firebaseApp.ts` file specifically and it has the same name as the test file in `next/`

Reply:
That's probably a good idea. Renamed to `firebaseCompat.test.ts`